pincer/core/http.py

```diff
@@ -9,6 +9,8 @@
 
 from aiohttp import ClientSession, ClientResponse
 
+# Im open for ideas on how to get __version__ without doing this
+import pincer
 from . import __package__
```

Comment:
On the same line as the `__package__` import
```suggestion
from . import __package__, __version__
```

Reply:
That causes a circular import